pkg/webui/console/views/gateway-overview/gateway-overview.js

```diff
@@ -58,6 +77,35 @@ export default class GatewayOverview extends React.Component {
     collaboratorsTotalCount: undefined,
   }
 
+  @bind
+  async getGlobalConf() {
+    if (this.state.globalConf) return
+
+    const { gtwId } = this.props
+
+    try {
+    const globalConf = await api.gateway.getGlobalConf(gtwId)
+    this.setState({ globalConf })
+    } catch(err) {
+      toast({
+        title: m.globalConfFailed,
+        message: err,
```

Comment:
With you current implementation of `getGlobalConf` this wont work since it doesnt unwrap errors. The object you are looking for is stored in `err.response.data`.